src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -96,6 +105,29 @@ const InnerWrapper = styled('div')`
   }
 `
 
+const Section = styled.div`
+  width: 100%;
+`
+
+const StartDateWrapper = styled.div`
+  position: relative;
+  margin-top: 0.375rem;
+  margin-bottom: 2rem;
+`
+
+const StartDateLabel = styled.p`
+  margin: 0 0 0.5rem 0.5rem;
+  font-size: 0.875rem;
+  line-height: 1;
+`
+
+const Heading = styled.h3`
+  margin: 0;
+  font-size: 2rem;
+  line-height: 2.5rem;
```

Comment:
```suggestion
  line-height: 1.25;
```

we try to have line-height set as a multiplier of the value. Referenced here https://www.notion.so/hedviginsurance/Styling-Documentation-4d829669ede84b1ba741609336231e74